deployment/cdk/opensearch-service-migration/lib/service-stacks/reindex-from-snapshot-stack.ts

```diff
@@ -113,19 +122,53 @@ export class ReindexFromSnapshotStack extends MigrationServiceCore {
             servicePolicies.push(getSecretsPolicy);
         }
 
+        const volumes = [sharedLogFileSystem.asVolume()];
+        const mountPoints = [sharedLogFileSystem.asMountPoint()];
+
+        // Calculate the volume size based on the max shard size
+        // Have space for the snapshot and an unpacked copy, with buffer
+        const planningSize = props.maxShardSizeGiB ?? 80;
+        const volumeSize = Math.max(
+            Math.ceil(planningSize * 2 * 1.15),
+            125 // Minimum volume size for ST1
+        )
+
+        // Volume we'll use to download and unpack the snapshot
+        const snapshotVolume = new ServiceManagedVolume(this, 'SnapshotVolume', {
+            name: 'snapshot-volume',
+            managedEBSVolume: {
+                size: Size.gibibytes(volumeSize),
+                volumeType: EbsDeviceVolumeType.ST1,
```

Comment:
Let's use GP3 here. I'm concerned the throughput and iops will be a bottleneck for small volume sizes. This will also enable us to remove the minimumVolumeSize above


> with a baseline throughput of 40 MB/s per TB
https://aws.amazon.com/ebs/volume-types/

Reply:
@AndreKurait Can you be more specific around the specific throughput parameters you expect the system to require?  There's also a significant cost difference.

Reply:
Discussed offline, given ST1s baseline throughput of `40MBps per TB` and uncertainty around throughput with volumes much smaller than 1TB, we've aligned to go with GP3 despite the ~2x cost increase (GP3 being comparable to Ephemeral Storage cost)